controllers/vpcendpoint/helpers.go

```diff
@@ -167,6 +227,12 @@ func (r *VpcEndpointReconciler) findOrCreateVpcEndpoint(resource *avov1alpha1.Vp
 		vpce = resp.VpcEndpoints[0]
```

Comment:
this seems a little brittle....add a nill check?

Reply:
Talked about this and there should always be at least one VpcEndpoint if we get to this point due to a previous check for

```go
if resp == nil || len(resp.VpcEndpoints) == 0 {
```